js/Model.elm

```diff
@@ -60,6 +61,7 @@ initModel : String -> Model
 initModel hostname =
     { stage = ReadyStage initReadyModel
     , hostname = hostname
+    , name = "Anonymous"
```

Comment:
We get a nontrivial loop space: Without edit it's `"Anonymous"` but if you edit and subsequently clear the name, it becomes `""`, even though the placeholder makes them believe it's still `"Anonymous"`.

Reply:
I think it's okay.